packages/slack-bot/src/utils.js

```diff
@@ -43,3 +44,32 @@ export function distancesFromEmbeddings({ queryEmbedding, embeddings }) {
   }))
   return distance
 }
+
+export async function downloadAudio(url, id) {
+  return new Promise((resolve, reject) => {
```

Comment:
why not just simply pipe the http response to the file using a file stream created e.g. with https://nodejs.org/api/fs.html#fscreatewritestreampath-options? I think it would simplify the implementation a bit

Reply:
Updated to address this point